docker-compose-test-e2e.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  todo-app:
+    container_name: todo-app-test-e2e
+    image: todo-app:test-e2e
+    network_mode: "host"
```

Comment:
Do we need this here? I think this skips having to bind ports - but do we need any networking?